src/main/controllers/helpers/AdminNotificationHelper.ts

```diff
@@ -57,7 +57,18 @@ export const getVisibleRequestFromAdmin = (
   return adminNotifications;
 };
 
-const isVisibleTribunalResponse = (response: TseRespondTypeItem) => {
+const isVisibleTribunalResponse = (response: TseRespondTypeItem, claimantResponseRequired: string) => {
+  if (claimantResponseRequired === YesOrNo.YES) {
+    if (response.value.isResponseRequired === YesOrNo.YES) {
```

Comment:
don't even need to check this second one. If response.value.isResponseRequired is NO then it'll go in on the next if

Reply:
I see what you're saying however I am going to leave this in for now. The next if statement is going to change to be more restrictive when I make requests that don't require a response disappear.